design/style.md

```diff
@@ -22,22 +24,20 @@ The DC/OS CLI is the main tool used by developers and operators to interact with
   * Show - Show a description or definition aka Describe, Get
   * List - List all objects
 
-### Arguments/Flags
+### Arguments/Options
+
+CLI arguments should follow the [GNU standards][1] as much as possible.
```

Comment:
Inline links would be more coherent with what we currently do in our `.md` files.

Reply:
I added them to the bottom as the line was quite long otherwise.